lambdas/format_input.py

```diff
@@ -84,8 +88,10 @@ def lambda_handler(event: dict, _context: dict) -> dict:
                 "the provided date and source, something likely went wrong."
             )
             return result
+
+        # NOTE: FEATURE FLAG: branching logic will be removed after v2 work is complete
         result["load"] = commands.generate_load_commands(
-            transform_output_files, run_type, source, timdex_bucket
+            transform_output_files, run_type, source, timdex_bucket, etl_version
```

Comment:
Same question here, re: pulling `ETL_VERSION` in the helper functions themslves vs parsing here and adding as a passed argument.

Reply:
Good question! Let's continue the conversation in this https://github.com/MITLibraries/timdex-pipeline-lambdas/pull/313#discussion_r1868395871!